docs/crds/avinfrasetting.md

```diff
@@ -122,6 +122,15 @@ AviInfraSetting CRD can be used to enable/disable Route Health Injection (RHI) o
 
 This overrides the global `enableRHI` flag for the virtualservices corresponding to the AviInfraSetting.
 
+#### Enable/Disable Public IP
+
+AviInfraSetting CRD can be used to enable/disable Public IP on the virtualservices created by AKO. 
+
+        network:
+          enablePublicIP: true
+
+Note that enabling Public IP is only supported for **public clouds**.
```

Comment:
is it supported for all public clouds or just AWS?

Reply:
It is for all public clouds.

Reply:
yes, for all public clouds